spec.bs

```diff
@@ -993,9 +995,11 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
   1. [=Assert=]: |trustedScoringSignalsURL|'s [=url/scheme=] is "`https`".
   1. Set |auctionConfig|'s [=auction config/trusted scoring signals url=] to
     |trustedScoringSignalsURL|.
-1. If |config|["{{AuctionAdConfig/maxTrustedScoringSignalsURLLength}}"]'s value is non-negative, set |auctionConfig|'s 
-  [=auction config/max trusted scoring signals url length=] to |config|["{{AuctionAdConfig/maxTrustedScoringSignalsURLLength}}"],
-  otherwise [=exception/throw=] a {{TypeError}}.
+1. If |config|["{{AuctionAdConfig/maxTrustedScoringSignalsURLLength}}"] [=map/exists=] and

```

Comment:
Same.
we don't throw when the field does not exist.
So you need to separate the if statements:
"
If ... [=map/exists=] {
if ... >=0, then set ..., otherwise throw ...
}
"